inspirehep/modules/disambiguation/logic.py

```diff
@@ -0,0 +1,428 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Inspire.
+# Copyright (C) 2016 CERN.
+#
+# Inspire is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Inspire is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Inspire; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Logic of Disambiguation module."""
+
+from __future__ import absolute_import, division, print_function
+
+from operator import add
+
+import celery
+from celery.utils.log import get_task_logger
+from flask import current_app
+
+from inspirehep.modules.disambiguation.records import create_author
+from inspirehep.modules.disambiguation.search import create_record, \
+    get_signature
```

Comment:
> Oooh, the recommended be:
> 
> ```python
> import (foo,
>     bar)
> ```
> right?

```python
import (
    foo,
    bar,
)
```
is better, because then you can add/remove imports and have cleaner Git histories.

> Imports using \ are following PEP-8.

Nope. From https://www.python.org/dev/peps/pep-0008/#maximum-line-length:

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

There's also https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses, and PEP8 explicitly mentions this line continuation invisible gotcha.

> Thanks, anyway I would appreciate straightforward comments.
Heh, I wasn't getting any answer before...